recipe/meta.yaml

```diff
@@ -10,7 +10,6 @@ source:
 
 build:
   number: 2
```

Comment:
I think it is better to bump the build number

Reply:
Thanks you!